extensions/amp-story/1.0/amp-story.js

```diff
@@ -328,6 +322,14 @@ export class AmpStory extends AMP.BaseElement {
 
     /** @private {?number} the timeout to show subscriptions dialog after delay */
     this.showSubscriptionsUITimeout_ = null;
+
+    /** @private {?number} the index of the page where the paywall would be triggered. */
+    this.subscriptionsPageIndex_ = this.storeService_.get(
```

Comment:
Getting this number will always be -1, it's not necessary to fetch it from the storeservice

Reply:
This is to potentially get the already resolved index from the store service, since the `amp-story-subscriptions` might already have initialized.